index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
         </ul>
       </div>
     </section>
-  </body>
-</html>
+    <section>
+        <div>
+         <div>
+             <h1 class="batu">
+                 My Recent <br> Works
+             </h1><hr>
```

Comment:
- [ ] The space between the `hr` tag and the first card is `62px`. Kindly implement it so that your design looks identical to that of the Figma template. 😄